hal/lib/athena/HALAthena.cpp

```diff
@@ -5,8 +5,6 @@
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
-#include "HAL/HAL.h"
```

Comment:
Moving that include is going to make the order checker complain because it's an "other include" that doesn't match the name of the file.